akka-cluster/src/main/scala/akka/cluster/AutoDown.scala

```diff
@@ -24,6 +23,24 @@ private[cluster] object AutoDown {
 }
 
 /**
+ * Used when no custom provider is configured and 'auto-down-unreachable-after' is enabled.
+ */
+final class AutoDowning(system: ActorSystem) extends DowningProvider {
+
+  private def clusterSettings = Cluster(system).settings
+
+  override def downRemovalMargin: FiniteDuration = clusterSettings.DownRemovalMargin
+
+  override def downingActorProps: Option[Props] =
+    clusterSettings.AutoDownUnreachableAfter match {
+      case d: FiniteDuration ⇒ Some(AutoDown.props(d))
+      case _ ⇒
+        // I don't think this can actually happen
+        throw new ConfigurationException("auto-down-unreachable-after must be finite if specified")
```

Comment:
it can if user configures `downing-provider-class = akka.cluster.AutoDowning` but doesn't specify `auto-down-unreachable-after`
The error message should probably be adjusted to capture that.